src/tests/ftest/io/nvme_pool_capacity.yaml

```diff
@@ -0,0 +1,48 @@
+server_manager:
+  srv_timeout: 500
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-C
+    - client-D
+    - client-E
```

Comment:
IOR 10 parallel job is getting run with 2 slots so total 20 process and that we can achieve with 2 clients. So may be we do not need Three clients.
This might help to change the Tag as well from large to medium as we can finish with Four systems